org.kde.krename.json

```diff
@@ -0,0 +1,60 @@
+{
+    "id": "org.kde.krename",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "krename",
+    "rename-icon": "krename",
+    "finish-args": ["--share=ipc", "--socket=fallback-x11", "--socket=wayland", "--filesystem=host" ],
```

Comment:
Those should be on their own lines

Reply:
what do you mean by "those"?

Reply:
Each element from the array in finish-args should be on its own line.